app/views/tag/show_for_author.html.erb

```diff
@@ -0,0 +1,43 @@
+<% if params[:controller] == "tag" || params[:controller] == "search" %>
```

Comment:
Was there not an efficient way to use the same `show.html.erb` template, and adapt it using one of these `if/else` statements for displaying posts by a single author?

Reply:
Perhaps anything that doesn't currently work with just one author, like the other tabs, could be disabled by checking the `params[:action]` to see if this is an instance of an author-specific tag show page and not just a normal one?

Reply:
I will try to change it